### PR TITLE
rework OWNERS [WIP]

### PR DESCRIPTION
**Pull request by goern**
:sparkles: rework OWNERS as mentioned in https://github.com/open-infrastructure-labs/ops-issues/issues/16

Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,18 +1,15 @@
...
+  - durandom
   - anishasthana
   - hemajv
-  - HumairAK
-  - tumido
   - martinpovolny
   - 4n4nd
+
+labels:
+  - sig/documentation
```
Comment (tumido) [inline OWNERS:15]: This PR is against the `operate-first/apps` root folder. I don't think `sig/documentation` is the right label. :) I think you can drop this bit here.

**Review by tumido — CHANGES_REQUESTED** [review]

File: OWNERS (hunk 1 above)
Comment (HumairAK) [reply OWNERS:15]: Looking at [this](https://github.com/kubernetes/kubernetes/blob/master/OWNERS#L33) it looks like you can apply labels to a file type regex. Maybe we can have one for *.md and `sig/documentation` 

I may not be fully understanding `filters` however.

Comment (HumairAK) [reply OWNERS:15]: Other cool ones would be if the `cluster-scope` is changed, a "cluster-scope` label or something similar would be nice, or if it's the odh path have it labelled `odh`

I could be getting carried away here.

Comment (tumido) [reply OWNERS:15]: I think we can also do this with separate `OWNERS` files per each sub folder or via [filters](https://www.kubernetes.dev/docs/guide/owners/#filters)

Comment (goern) [reply OWNERS:15]: all the things said are right and good, feel free to open a PR against this PR ;)

**Comment by goern** [comment]
BTW... I just figured this is the wrong repo ;) we said we gonna do it on `support/` :facepalm:

**Comment by anishasthana** [comment]
So from some discussion, we settled on the following list for the file:

OWNERS:
Anand, Humair, Tom
Reviewers:
OWNERS + Anish, Hema, Harshad, Martin

PRs need 2 Reviews followed by 1 approval

**Comment by anishasthana** [comment]
We can close this PR, https://github.com/operate-first/apps/pull/162 meets the same requirement.

**Pull request closed without merging**